### PR TITLE
feature/update_removed_devicehealthscripts

### PR DESCRIPTION
**Pull request by dawit-simeon**

### CODE REVIEW

File: baseline.md [hunk 1]
```diff
@@ -1169,42 +1172,6 @@ Used to manage authorization related settings across the company.
 | What is the end-user impact? | N/A |
 | Learn more | [Microsoft Intune and Azure Log Analytics](https://techcommunity.microsoft.com/t5/device-management-in-microsoft/microsoft-intune-and-azure-log-analytics/ba-p/463145) |
 
-## Office 365 > Exchange > AdminAuditLogConfig
-*PowerShell/Exchange/Organization/AdminAuditLogConfig*
-
-###### hidden-header
-
-### Admin Audit Log Settings
```
Comment (jeff-simeon) [inline baseline.md:1177]: why was this removed?

Comment (rdorta-simeon) [reply baseline.md:1177]: We removed those configs from the baseline

**Review by jeff-simeon — APPROVED** [review]

**Review by david-simeon — APPROVED** [review]
